lib/web_ui/lib/src/engine/semantics/accessibility.dart

```diff
@@ -50,6 +51,10 @@ void initializeAccessibilityAnnouncements() {
   });
 }
 
+/// Duration for which a live message will be present in the DOM for the screen
+/// reader to announce it.
+const Duration liveMessageDuration = Duration(milliseconds: 300);
```

Comment:
Instead of making this a `const` can this be mutable so that tests can set a smaller duration than 300ms? I don't know how many tests we currently have, but if the union of all tests have to await on this 10 times, it lengthens the runtime of the test by 3 seconds. I think we could use 10 times smaller delays in the tests.

Reply:
Done!